### PR TITLE
Update document selector

### PR DESCRIPTION
**Pull request by tahmeedshah**

### CODE REVIEW

File: app/views/forms/edit/bch/directives/edit-modified-organism.directive.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ import 'views/forms/directives/view-terms-hierarchy';
 
 					var queryOptions = {
 						realm     : realm.value,
-						fieldQueries: ['schema_s:contact AND type_s:person'],
```
Comment (blaisefonseca) [inline app/views/forms/edit/bch/directives/edit-modified-organism.directive.js:44]: why `type_s:person` was removed?

File: app/views/forms/edit/directives/edit-contact.directive.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ function($http, $filter, $rootScope, $location, $q, storage, roleService, thesau
             $scope.onContactQuery = function(searchText){
                 
                 var queryOptions = {
-                    fieldQueries: ['schema_s:contact AND type_s:organization'],
+                    fieldQueries: ['schema_s:contact AND (refReferenceRecords_ss:* OR refNationalRecords_ss:*)'],
```
Comment (blaisefonseca) [inline app/views/forms/edit/directives/edit-contact.directive.js:46]: why type_s:organization was removed?

**Review by blaisefonseca — CHANGES_REQUESTED** [review]
please maintain the existing contact schema query

File: app/views/forms/edit/bch/directives/edit-modified-organism.directive.js (hunk 1 above)
Comment (tahmeedshah) [reply app/views/forms/edit/bch/directives/edit-modified-organism.directive.js:44]: As per DEVDESK-798:
add below query for contacts
schema_s:contact AND (refReferenceRecords_ss:* OR refNationalRecords_ss:*)

Comment (blaisefonseca) [reply app/views/forms/edit/bch/directives/edit-modified-organism.directive.js]: it was an example, we need to keep existing query

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**